code/modules/games/tarot.dm

```diff
@@ -0,0 +1,41 @@
+/* this is a playing card deck based off of the Rider-Waite Tarot Deck.
+*/
+
+/obj/item/weapon/deck/tarot
+	name = "deck of tarot cards"
+	desc = "For all your occult needs!"
+	icon_state = "deck_tarot"
+
+/obj/item/weapon/deck/tarot/New()
+	..()
+
+	var/datum/playingcard/P
+	for(var/name in list("Fool","Magician","High Priestess","Empress","Emperor","Hierophant","Lovers","Chariot","Strength","Hermit","Wheel of Fortune","Justice","Hanged Man","Death","Temperance","Devil","Tower","Star","Moon","Sun","Judgement","World"))
```

Comment:
I feel vaguely uneasy about shadowing `src.name` here but it's not actually important